.zuul.yaml

```diff
@@ -1,23 +0,0 @@
-- project:
-    check:
-      jobs:
-        - "thoth-coala"
-        - "thoth-pytest"
```

Comment:
I think @goern mentioned to remove only `thoth-pytest` for now :)

Reply:
you are right, absolutely my bad! will recover the file and fix this

Reply:
No problem, thanks :smiley:

Reply:
recovered and removed

Reply:
fixed, now the pytest job should not be present